pulp/Containerfile

```diff
@@ -21,7 +21,8 @@ RUN pip3 install --upgrade \
   pulp-maven${PULP_MAVEN_VERSION} \
   pulp-python${PULP_PYTHON_VERSION} \
   pulp-rpm${PULP_RPM_VERSION} \
-  requests
+  requests &&
```

Comment:
```suggestion
  requests \
```

Reply:
Thank you, you need both actually. I fixed it.